cpp/tests/sssp/sssp_test.cu

```diff
@@ -397,34 +391,48 @@ class Tests_SSSP : public ::testing::TestWithParam<SSSP_Usecase> {
         }
       }
     }
-
-    // Done with device mem. Free it
-    cudaStream_t stream{nullptr};
-    ALLOC_FREE_TRY(col_src.data, stream);
-    ALLOC_FREE_TRY(col_dest.data, stream);
-    ALLOC_FREE_TRY(col_weights.data, stream);
   }
 };
 
 std::vector<double> Tests_SSSP::SSSP_time;
 
-TEST_P(Tests_SSSP, CheckFP32_DIST_NO_PREDS) {
-  run_current_test<int, int, float, true, false>(GetParam());
+TEST_P(Tests_SSSP, CheckFP32_NO_RANDOM_DIST_NO_PREDS) {
```

Comment:
Can we add a comment on what is "random" or not in these tests?

Reply:
Added comments above definition of `run_current_test` in https://github.com/rapidsai/cugraph/pull/764/commits/1528aebe9ea39d6ac0060b1068a76f8c2a36056c